parts/language/lang_en.lua

```diff
@@ -104,36 +104,36 @@ return{
 	chatStart="------Beginning of log------",
 	chatHistory="------New messages below------",
 
-	noRooms="No Rooms Now",
-	roomsCreateFailed="Failed to create room",
-	roomsFetchFailed="Failed to fetch rooms",
+	noRooms="Such emptiness much void, come back later?",
```

Comment:
Are we supposed to meme in a non-meme language?

Reply:
The idea is to convey in a more lighthearted and flavorful manner, instead of b l a n d

Reply:
OK then, this one is a personal taste thing. I prefer straight forward text for the UI, and keep memes and jokes to where they are meant to be (eg, easter eggs, MotDs, story texts).